docs/using/event-types.md

```diff
@@ -44,6 +44,83 @@ The custom schema for an event type can be as simple as
 `{ "\additionalProperties\": true }` to allow arbitrary JSON, but will usually
 have a more specific definition :)
 
+### Compatibility modes
+
+Compatibility modes are used to control schema changes. Each mode
+solves a specific problem and thus present different constraints. It
+ranges from being very constraining, allowing just some minor changes
+to allowing everything.
+
+By default, Nakadi uses the forward mode, which is further explained
+bellow, together with the other two modes: compatible and none.
+
+#### Forward
+
+This mode has been designed to allow event type owners to change their
+schemas without breaking existing consumers.
+
+Its called forward because using older schemas it's safe to read
+events generated by newer schemas. On the other hand, it's not
+backward, in the sense that trying to read an older event using a
+newer schema is not safe.
+
+So if the consumer has been developed with schema version `1.5.0` in
+mind, reading events from `1.5.0` on is safe, for example, `1.6.0`. But
+reading event generated by older schemas, like `1.4.0` is not safe.
+
+Supported changes:
+
+1. changes to meta attributes: titles, descriptions and examples.
+2. addition of new attributes.
+3. marking attributes as required.
+
+Consumers reading events from forward compatible event types SHOULD
+ignore event attributes not declared in the event type schema.
+
+This mode is default because it solves the most common requirement:
+guarantee stability of deployed consumer applications when schema
```

Comment:
`guarantee stability` -> `guarantee the stability`
`consumer` -> `consumers`
`when schema` -> `when the schema`